config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml

```diff
@@ -94,13 +94,39 @@ spec:
                   deactivation:
                     description: Keeps parameters concerned with user deactivation
                     properties:
+                      UserSignupUnverifiedRetentionDays:
```

Comment:
because I pushed the `config/` directory a few days ago and the CRDs have changed since then. The current logic that is in api repo right now doesn't update this directory - it updates only the `deploy/olm-catalog/` one. 
I made the PR with the `config/` directory separately to make this PR a bit smaller. https://github.com/codeready-toolchain/host-operator/pull/477

Reply:
I'm sorry but I lost track in your explanations: could you elaborate on `config` vs `deploy/olm-catalog`, please?